### ISSUE TITLE
Which existing OSM compounds should we send for biofunctionalisation?

### ISSUE THREAD

**Issue by david1597** [OpenSourceMalaria/OSM_To_Do_List#529]
Scott Obach (Pfizer) has obtained some interesting and promising results from the late stage functionalisation through biosynthesis of some of the series 4 compounds (#513).

Following these results, we should think about which other existing OSM compounds it would be useful to send to Scott for similar biofunctionalisation.

**Comment by mattodd** [comment]
The most relevant structures are I think those in #400 and #484 - i.e. the best overall compounds and those for which we in Sydney are most likely to have physical samples. Choices choices.

**Comment by MFernflower** [comment]
MMV688895
MMV669000
MMV897707

are a few that I'd think would be interesting @mattodd

**Comment by mattodd** [comment]
OK, so Scott refreshed my memory that we need to send 2 mg of each compound to him. So what'll be the final list here @david1597 ? i.e. based on what's desirable and what we have in stock? Could you post a pic?

Note that, by email, Scott mentioned that he has extensive characterisation (including 2D NMR; (data will be transferred)) of the active compounds, meaning he has a high level of certainty that we have a para-OH group. i.e. we're certainly interested in other compounds with the possibility of that particular modification.

**Comment by david1597** [comment]
How about these four compounds? They're from the "frontrunners", MMV68896 and MMV663915 both allow for the para-OH modification and are closely related to the compound in #513 that successfully went through biosynthesis. 

![aug17 compounds to send for biofunctionilisation](https://user-images.githubusercontent.com/30584472/29196630-f3d88fe6-7e78-11e7-97b8-ad8e36f2f70e.jpg)

**Comment by MFernflower** [comment]
I am very curious to see what the pyridine containing compounds (MMV670944/670963) will do - will they form n-oxides? will they hydroxylate and become even more potent?

**Comment by david1597** [comment]
MMV670944 was examined in the previous biosynthesis -  #513.

**Comment by MFernflower** [comment]
Mistake aside, I would be curious to see what happens to mmv669848 (the isoindole) as I bet it will easily take a OH on the benzene portion of the isoindole @david1597

**Comment by david1597** [comment]
The isoindole would be a good addition. I'll just need to check we actually have some of that compound.

**Comment by david1597** [comment]
Unfortunately, we don't have any of the isoindole compound in stock.

**Comment by mattodd** [comment]
OK, we should decide and ship to Scott. Suggestions:

_Pretty obvious_
MMV688896 (OSM-S-279)
MMV663915 (OSM-S-293)

Negative control might be useful, where the oxidized position is blocked, e.g. MMV672687 (OSM-S-390)

![follow up pfizer compounds](https://user-images.githubusercontent.com/4386101/29821259-9f9110c0-8d0a-11e7-91c3-c32e57890198.png)

_Other suggestions made:_
MMV639565 (David) similar idea to OSM-S-390, so one or the other not both.
MMV663915 (David) not terribly soluble to begin with, so perhaps no great advantage over the original Pfizer compound (i.e. MMV693155)
MMV688895 (Mandrake) contains northeast Pyr, so likely goes to N-oxide 
MMV669000 (Mandrake) contains isoindole. Highly inactive compound (>10 uM), so a long shot. MMV669848 - we’re out.
MMV897707 (Mandrake) NMe2 substitution, but also para-blocked by F.

Instead, how about a _caged, non-aromatic compound_, like MMV1557938 (OSM-S-398) (or the cubane MMV897700 at 0.37 uM)?

The questions we’re asking here are: is it generally the case that hydroxylation in that aromatic ring helps? What happens when that position is blocked? Can hydroxylation take place in that region of the molecule on an aliphatic position?

Do these suggestions tally with what you were thinking @david1597 ? And do we have samples?

([files](https://au-mynotebook.labarchives.com/share/Matthew%2520Houghton%2520Todd/MjkuOTAwMDAwMDAwMDAwMDAyfDE1NC8yMy9UcmVlTm9kZS8yMDMyNTc4Mzc2fDc1Ljg5OTk5OTk5OTk5OTk5))

**Comment by MFernflower** [comment]
My vote is for the LHS cubane

**Comment by david1597** [comment]
Agreed. How about the following four compounds then? We have sample available for all.

![aug17 compounds to send for biofunctionilisation2](https://user-images.githubusercontent.com/30584472/29855041-a1aa29b4-8d8c-11e7-8ef2-7d1e4eab5bdb.jpg)

**Comment by drc007** [comment]
This paper might be of interest. https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5579547/pdf/ao7b01054.pdf

**Comment by david1597** [comment]
The following four compounds were shipped on 18 Sept.

![aug17 compounds to send for biofunctionilisation2](https://user-images.githubusercontent.com/30584472/30526253-eb1e33c4-9c5a-11e7-9671-60aabde15170.jpg)

Results will be posted in the [ELN](https://au-mynotebook.labarchives.com/share/David%2520Smith%2520OSM/OTMuNjAwMDAwMDAwMDAwMDF8MzAxNzEvNzIvVHJlZU5vZGUvMzY1MTg4NDE5OXwyMzcuNg==
) when they are ready.

As we have now decided which compounds should be shipped and they are on their way, I am closing this issue. A new issue will be created to discuss any interesting results.
